docs/rst/tutorials/core/deployment/ds_wan_tcp/ds_wan_tcp.rst

```diff
@@ -255,8 +281,10 @@ Run the example:
 
 .. note::
 
-    The requirements to achieve TCP communication over WAN with Discovery Server as EDP in a real deployment are simply launching the three elements of the communication (``talker``, ``listener`` and discovery *server*) with their corresponding XML configurations applied.
-    To do so in a real scenario, setting the proper firewall or router configuration rules should be considered.
+    The requirements to achieve TCP communication over WAN with Discovery Server as EDP in a real deployment are launching the three elements of the communication (``talker``, ``listener`` and discovery *server*) with their corresponding XML configurations applied, and setting the proper firewall or router configuration rules.
+    This should be considered only for the discovery *server* and each ``talker`` node.
```

Comment:
I think it would be worth being a little more specific here. We could add something like it should be necessary to configure one port forwarding rule for the server, and another port per every two clients communicating over the WAN (in either one of the sides).